test.adoc

```diff
@@ -0,0 +1,7 @@
+Don't do this.
+
+You can certainly do this.
+
+However, never forget how great OpenShift Container Platform is.
```

Comment:
Use the AsciiDoc attribute '{product-title}' rather than the plain text product term 'OpenShift Container Platform', unless your use case is an exception.